spyder/plugins/editor/plugin.py

```diff
@@ -85,6 +85,7 @@ class Editor(SpyderPluginWidget):
     breakpoints_saved = Signal()
     run_in_current_extconsole = Signal(str, str, str, bool, bool)
     open_file_update = Signal(str)
+    debugging_file_msg = Signal()

```

Comment:
We have for some time now an "unwritten rule" of naming signals starting with the `sig_` prefix and trying to use a name that is like, saved, requested, moved, etc.

So in this case, I suggest we use something like:

`sig_file_debug_message_requested = Signal()`

I know it is a bit long but the idea of having the `requested` in there is that the widget that emits this thing cannot actually perform the action but requests the action to someone else listening to that signal.